paddle/fluid/inference/tests/api/CMakeLists.txt

```diff
@@ -735,6 +763,7 @@ if(WITH_MKLDNN)
     set_tests_properties(test_analyzer_quant_performance_benchmark PROPERTIES TIMEOUT 120)
     set_tests_properties(test_analyzer_int8_mobilenetv2 PROPERTIES TIMEOUT 120)
     set_tests_properties(test_analyzer_int8_mobilenetv1 PROPERTIES TIMEOUT 120)
+    set_tests_properties(test_analyzer_int8_mobilenetv3_large PROPERTIES TIMEOUT 120)
```

Comment:
I think I read in somebody's PR that TIMEOUT is not necessary if it does not take too much time @sfraczek.
Could you please confirm in the team ? If this line does not exist, will the test pass ?

Reply:
To make the test fast you may change the bs to 50 maybe

Reply:
This test test_analyzer_int8_mobilenetv3_large is quite slow because it tests FP32, calculate scales for PTQ and test INT8 and it had timeout in Coverage-CI that is why I extended timeout value.

Reply:
I think we can extend timeout to 300 like in examples above this. Will it be enough?

Reply:
This set of 120 is already enough.